src/main/java/org/gaul/s3proxy/S3ProxyHandlerJetty.java

```diff
@@ -50,6 +54,13 @@
                 ignoreUnknownHeaders, corsAllowAll, servicePath);
     }
 
+    private void sendS3Exception(HttpServletRequest request,
```

Comment:
Added `static`, although I have an afterthought: do we want `static` even though the method accesses `handler` which is an instance member?